x-pack/plugins/uptime_monitoring/public/apps/start_app.tsx

```diff
@@ -4,26 +4,21 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiPage, EuiPageContent, EuiTitle } from '@elastic/eui';
-import React from 'react';
 import 'react-vis/dist/style.css';
 import 'ui-bootstrap';
 import 'ui/autoload/all';
 import 'ui/autoload/styles';
+import chrome from 'ui/chrome';
 import 'ui/courier';
 import 'ui/persisted_log';
 import 'uiExports/autocompleteProviders';
+import { createApolloClient } from '../lib/adapters/framework/apollo_client_adapter';
 import { UMFrontendLibs } from '../lib/lib';
+import { UptimeMonitoringApp } from '../uptime_monitoring_app';
 
 export async function startApp(libs: UMFrontendLibs) {
-  libs.framework.render(
-    <EuiPage>
-      <EuiPageContent>
-        <EuiTitle>
-          <h2>Uptime Monitoring</h2>
-        </EuiTitle>
-        <p>This is where the Uptime app will live.</p>
-      </EuiPageContent>
-    </EuiPage>
-  );
+  const uriPath = `${chrome.getBasePath()}/api/uptime_monitoring/graphql`;
+  const xsrfHeader = chrome.getXsrfToken();
```

Comment:
Using chrome directly vs via your framework adapter might be a leaky abstraction? your call

Reply:
Yeah - I agree, that should be added in the framework adapter. Will update.